pkg/util/etcdutil/etcdutil.go

```diff
@@ -23,12 +24,17 @@ import (
 	"golang.org/x/net/context"
 )
 
-func ListMembers(clientURLs []string) (*clientv3.MemberListResponse, error) {
+func NewClient(clientURLs []string, tc *tls.Config) (*clientv3.Client, error) {
```

Comment:
i do not think it worth the effort to add this indirection (this NewClient func). just pass in tls in cfg?